javarosa/src/test/java/org/javarosa/xpath/test/XPathEvalTest.java

```diff
@@ -68,6 +68,8 @@ private void testEval(String expr, FormInstance model, EvaluationContext ec, Obj
                     fail("Doubles outside of tolerance [" + o + "," + t + " ]");
                 } else if (Double.isNaN(o) && !Double.isNaN(t)) {
                     fail("Result was NaN when not expected");
+                } else if (Double.isNaN(t) && !Double.isNaN(o)) {
```

Comment:
just added this because I realized we were not actually checking for this failure case... @ctsims Are there other test files where I might want to check that we were overlooking this?